corehq/apps/locations/tests/test_bulk_management.py

```diff
@@ -0,0 +1,110 @@
+from django.test import TestCase
+from corehq.apps.domain.shortcuts import create_domain
+from corehq.apps.locations.bulk_management import bulk_update_organization
+
+# These example types and trees mirror the information available in the upload files
+
+# TODO What fields should be editable in the upload?
```

Comment:
Nevermind, it's in the [spec](https://docs.google.com/document/d/1gZFPP8yXjPazaJDP9EmFORi88R-jSytH6TTgMxTGQSk/edit#heading=h.lxq3ebm9juwn)